### PR TITLE
Trees

### PR DESCRIPTION
**Pull request by catdude2000**

### CODE REVIEW

**Comment by catdude2000** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [ ] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: javascript/linked-list/trees/trees.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    // go left
+    if (node.left) {
+      this.preOrder(node.left);
+    }
+    // go right
+    if (node.right) {
+      this.preOrder(node.right);
+    }
+    return;
+  }
```
Comment (DSFrey) [inline javascript/linked-list/trees/trees.js:27]: The challenge asks that you return an array of values in the correct order, but you are not returning anything. Also, the initial call should require no arguments (we know it will always start its traversal from the root).

File: javascript/linked-list/trees/trees.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+class BinarySearchTree extends BinaryTree {
+  constructor(root = null) {
+    super(root);
+  }
+  //if no root, num is now root
+  add(number, node) {
```
Comment (DSFrey) [inline javascript/linked-list/trees/trees.js:35]: Again, our initial call should only have a single argument (the value we are adding).
